nilearn/maskers/surface_labels_masker.py

```diff
@@ -361,29 +412,44 @@ def fit_transform(self, img, y=None):
         del y
         return self.fit().transform(img)
 
-    def inverse_transform(self, masked_img):
-        """Transform extracted signal back to surface object.
+    def inverse_transform(self, signals):
```

Comment:
good catch: we also need to rename the parameter in inverse_transforl in SurfaceMasker

most masker either call it: X, signals or region_signals

would be a nice thing to standardize...